pkg/console/subresource/deployment/deployment_test.go

```
@@ -525,16 +533,17 @@ func TestDefaultDeployment(t *testing.T) {
-		authnConfig           *configv1.Authentication
+		deployment               *appsv1.Deployment
+		consoleConfigMap         *corev1.ConfigMap
+		serviceCAConfigMap       *corev1.ConfigMap
+		authServerCAConfigMap    *corev1.ConfigMap
+		trustedCAConfigMap       *corev1.ConfigMap
+		oAuthClientSecret        *corev1.Secret
+		sessionSecret            *corev1.Secret
+		consoleServingCertSecret *corev1.Secret
+		proxyConfig              *configv1.Proxy
+		infrastructureConfig     *configv1.Infrastructure
+		authnConfig              *configv1.Authentication
```

golangci-lint (2.11.4)

[error] field authnConfig is unused

(unused)